airflow/providers/openlineage/utils/utils.py

```diff
@@ -269,6 +269,25 @@ class TaskInfo(InfoJsonEncodable):
     }
 
 
+class TaskInfoComplete(TaskInfo):
+    """Defines encoding BaseOperator/AbstractOperator object to JSON used when user enables full task info."""
+
+    includes = []
+    excludes = [
+        "_BaseOperator__instantiated",
+        "_dag",
+        "_hook",
+        "_log",
+        "_outlets",
+        "_inlets",
+        "_lock_for_execution",
+        "handler",
+        "params",
+        "python_callable",
+        "retry_delay",
```

Comment:
I think we can remove some (suggestion below) or even all stuff from here. It'll not be serialized anyway, so we'll get e.g. `python_callable:"<<non-serializable: function>>"`. No strong opinion on this one, but I don't see why we should exclude anything if it's turned on explicitly now.
```suggestion
        "_BaseOperator__instantiated",
        "_dag",
        "_hook",
        "_log",
        "_outlets",
        "_inlets",
        "_lock_for_execution",
```

Reply:
Yeah, but do we gain something by adding those? They were added after checking if it makes sense - the point for this PR is to enable serialization for those that we _don't know_ if are okay. I think there's no point for that `python_callable` there.